### PR TITLE
Add random delays in worker functions

### PR DESCRIPTION
**Pull request by swarnim-vyas**
This PR adds flag to introduce random delays in worker functionalities

### CODE REVIEW

File: worker.py [hunk 1]
```diff
@@ -63,15 +75,25 @@ def execute_task(self, task: Task, future: Future):
...
 
     def sigterm_handler(self, signum, frame):
         logging.info("Exiting gracefully.")
         sys.exit(0)
 
+    def add_random_delay(self):
+        if self.add_delay:
+            if randrange(100) > 50:
+                logging.info(f"AddDelay is true - Adding delay of 5 seconds.")
+                time.sleep(5)
```
Comment (krish8484) [inline worker.py]: randomize the delay time as well?

File: README.md [hunk 2]
```diff
@@ -15,5 +15,9 @@ Steps to Run
 5) Start the scheduler server from one terminal - python3 scheduler_server.py
 6) Start the worker server in another terminal - python3 worker_server.py <PortNumber>
     # Example Command: python3 worker_server.py 50054
+    # Example Command: python3 worker_server.py 50054 --addDelay
```
Comment (krish8484) [inline README.md]: lets not use # here, it breaks the font of the md file

File: worker.py (hunk 1 above)
Comment (swarnim-vyas) [reply worker.py]: Randomized

File: README.md (hunk 2 above)
Comment (swarnim-vyas) [reply README.md]: Removed

**Review by aadhithNarayan — APPROVED** [review]

**Review by krish8484 — APPROVED** [review]

**Pull request merged**